cluster/hook-template.yaml

```diff
@@ -42,7 +42,7 @@ objects:
           terminationGracePeriodSeconds: 180
           containers:
           - name: hook
-            image: ${REGISTRY}/${DOCKER_REPO}/hook:${IMAGE_TAG}
+            image: ${REGISTRY}/${DOCKER_REPO}-hook:${IMAGE_TAG}
```

Comment:
I don't think this is correct now. What is the value here for the generated `yaml` file?